Content.Client/Administration/UI/Tabs/ObjectsTab/ObjectsTab.xaml.cs

```diff
@@ -174,6 +163,17 @@ private void HeaderClicked(ObjectsTabHeader.Header header)
         RefreshObjectList();
     }
 
+    private string GetLocalizedEnumValue(ObjectsTabSelection selection)
+    {
+        return selection switch
+        {
+            ObjectsTabSelection.Grids => Loc.GetString("object-tab-object-type-grids"),
+            ObjectsTabSelection.Maps => Loc.GetString("object-tab-object-type-maps"),
+            ObjectsTabSelection.Stations => Loc.GetString("object-tab-object-type-stations"),
+            _ => throw new ArgumentOutOfRangeException(nameof(selection), selection, null),
+        };
+    }
+
```

Comment:
You can just .ToString().ToLower() the enum but I don't think this is a huge deal atm.